chapters/03_queueing_model/main.tex

```diff
@@ -0,0 +1,252 @@
+study queues.
+Under the assumption that all rates (arrival and service) are Markovian the
+queueing system can be represented by a Markov chain
+model~\cite{kemeny1976markov}.
+The states of the Markov chain are denoted by \((u,v)\) where:
+
+\begin{itemize}
+    \item \(u\) is the number of individuals blocked in waiting zone 2
+    \item \(v\) is the number of individuals either in waiting zone 1 or in the
+    service centre
+\end{itemize}
+
+The set of all possible combination of pairs \((u, v)\) form all the possible 
+states that the system can visit.
+We denote the state space of the Markov chain as the set \(S=S(T)\) which can be
+written as the disjoint union:
+
+\begin{align}
+    S(T) =& S_1(T) \cup S_2(T) \text{ where:} \nonumber \\
+    S_1(T) =& \left\{(0, v)\in\mathbb{N}_0^2 \; | \; v < T \right\}
+    \label{eq:definition_of_S_as_disjoint_union} \\
+    S_2(T) =& \{(u, v)\in\mathbb{N}_0^2 \; | \; v \geq T \} \nonumber
+\end{align}
+
+\(S_1\) consists of the set of states where the number of individuals in waiting
+zone 1 is less than \(T\) (i.e. \(v < 0\)) and subsequently the number of
+individuals in waiting zone 2 is zero (i.e. \(u = 0\)).
+Similarly, \(S_2\) consists of the set of states where the number of individuals
+in waiting zone 1 is greater than or equal to \(T\) (i.e. \(v \geq T\)) and 
+hence it is possible for individuals to be at waiting zone 2 (i.e. 
+\(u \geq 0\)).
+
+Having defined the set of states of the Markov chain model, the generator
+matrix can also be obtained.
+The generator matrix \(Q\) of the Markov chain consists of the
+rates between the numerous states of the model.
+Every entry \( Q_{ij} = Q_{(u_i, v_i),(u_j, v_j)} \) represents the rate from
+state \( i = (u_i, v_i) \) to state \( j = (u_j , v_j) \) for all
+\( (u_i, v_i), (u_j, v_j) \in S \).
+The entries of \(Q\) can be calculated using the state-mapping function
+described in (\ref{eq:markov_transition_rate}):
+
+\begin{equation} \label{eq:markov_transition_rate}
+    Q_{ij} =
+    \begin{cases}
+        \Lambda, & \textbf{if } (u_i, v_i) - (u_j, v_j) = (0,-1) \textbf{ and }
+        v_i < \text{t} \\
+        \lambda_1, & \textbf{if } (u_i, v_i) - (u_j, v_j) = (0,-1)
+        \textbf{ and } v_i \geq \text{t} \\
+        \lambda_2, & \textbf{if } (u_i, v_i) - (u_j, v_j) = (-1,0) \\
+        v_i \mu, & \textbf{if } (u_i, v_i) - (u_j, v_j) = (0,1) \textbf{ and }
+        v_i \leq C \textbf{ or} \\ & \hspace{0.37cm}(u_i, v_i) - (u_j, v_j) =
+        (1,0) \textbf{ and } v_i = T \leq C \\
+        C \mu, & \textbf{if } (u_i, v_i) - (u_j, v_j) = (0,1) \textbf{ and }
+        v_i > C
+        \textbf{ or} \\ & \hspace{0.37cm}(u_i, v_i) - (u_j, v_j) = (1,0)
+        \textbf{ and } v_i = T > C\\
+        -\sum_{j=1}^{|Q|}{Q_{ij}} & \textbf{if } i = j \\
+        0, & \textbf{otherwise}
+    \end{cases}
+\end{equation}
+
+Note that \(\Lambda\) here denotes the overall arrival rate in the model by both
```

Comment:
Can you add something here describing a function that maps a state to all possible starts that it can transition to with non zero rate.

Reply:
Will do